src/components/NavIcons.js

```diff
@@ -54,43 +73,76 @@ const IconWrapper = styled.div`
   display: grid;
   grid-template-columns: 0.15fr 0.15fr 0.15fr 0.15fr 0.15fr;
   grid-template-rows: 1fr 1fr 1fr;
-  row-gap: 30%;
+  row-gap: 10%;
   fill: var(--main-red);
   justify-content: center;
-  margin-top: 100px;
+  margin-top: 75px;
+  color: var(--main-white);
+  font-size: 18px;
```

Comment:
Better use em or rem for accessibility reasons.

Reply:
You are right, but this is something I wanna do after the video.